src/main/java/uk/gov/ch/oauth/UserProfileResponse.java

```diff
@@ -48,13 +60,22 @@ public void setSurname(String surname) {
 
     @JsonProperty("permissions")
     @Override
-    public void setPermissions(Map<String, Boolean> permissions) {
+    public void setPermissions(final Map<String, Boolean> permissions) {
```

Comment:
Strange that you've only done this on this setter. Could you explain? When these methods are just wrappers should we mimic the superclasses finalisations?

Reply:
We don't need to provide annotations to the getters as we don't send the values over the wire.